translations/de/Info/task_description.html

```diff
@@ -2,7 +2,7 @@
      style="width: 300px;float: right; margin-top: 30px; padding: 0 10px 0 30px;"/>
 
 <p>
-    Erinnerst du dich an<a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis"> die Basiszahl</a> und
+    Erinnerst du dich an <a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis">die Basiszahl</a> und 
```

Comment:
Hi Dahlia

The rule is you can write it in capital letters when writing personal to someone, and yes it's recommended. I'm sorry I just used the first CheckiO Elementary Lesson (Buzz Fuzz) as reference, and there it is lower-case! 
But yeah, no problemo, we should change these, it's more informal.

howdy

stefan

> On 10 Sep 2016, at 23:14, DahliaSR <notifications@github.com> wrote:
> 
> In translations/de/Info/task_description.html:
> 
> > @@ -2,7 +2,7 @@
> >       style="width: 300px;float: right; margin-top: 30px; padding: 0 10px 0 30px;"/>
> >  
> >  <p>
> > -    Erinnerst du dich an<a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis"> die Basiszahl</a> und
> > +    Erinnerst du dich an <a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis">die Basiszahl</a> und 
> Should read as: Erinnerst Du Dich an...
> 
> I recommend to use Rechtschreibprüfung Online by Duden.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>